src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -95,6 +102,20 @@ class IOUAmountPage extends React.Component {
         this.focusTextInput();
     }
 
+    componentWillUnmount() {
+        if (!this.unsubscribeAppStateSubscription) {
+            return;
+        }
+        this.unsubscribeAppStateSubscription();
```

Comment:
This function does not return a method to unsubscribe...

https://reactnative.dev/docs/appstate#addeventlistener

Reply:
`this.unsubscribeAppStateSubscription.remove()`

Reply:
Ah yes, good catch. I am seeing some console errors for this. @akshayasalvi can you please fix this?

Reply:
Hi,
Okay sure. I'll do that.

Reply:
No worries, we've fixed it already. This was taken care of in https://github.com/Expensify/App/pull/6909

Reply:
Thanks for the help